src/service.rs

```diff
@@ -554,23 +540,23 @@ mod tests {
 
     #[test]
     fn test_service_request() {
-        let req = TestRequest::default().to_srv_request();
-        let (r, pl) = req.into_parts();
-        assert!(ServiceRequest::from_parts(r, pl).is_ok());
-
-        let req = TestRequest::default().to_srv_request();
-        let (r, pl) = req.into_parts();
-        let _r2 = r.clone();
-        assert!(ServiceRequest::from_parts(r, pl).is_err());
-
-        let req = TestRequest::default().to_srv_request();
-        let (r, _pl) = req.into_parts();
-        assert!(ServiceRequest::from_request(r).is_ok());
-
-        let req = TestRequest::default().to_srv_request();
-        let (r, _pl) = req.into_parts();
-        let _r2 = r.clone();
-        assert!(ServiceRequest::from_request(r).is_err());
+        // let req = TestRequest::default().to_srv_request();
+        // let (r, pl) = req.into_parts();
+        // assert!(ServiceRequest::from_parts(r, pl).is_ok());
+
+        // let req = TestRequest::default().to_srv_request();
+        // let (r, pl) = req.into_parts();
+        // let _r2 = r.clone();
+        // assert!(ServiceRequest::from_parts(r, pl).is_err());
+
+        // let req = TestRequest::default().to_srv_request();
+        // let (r, _pl) = req.into_parts();
+        // assert!(ServiceRequest::from_request(r).is_ok());
+
+        // let req = TestRequest::default().to_srv_request();
+        // let (r, _pl) = req.into_parts();
+        // let _r2 = r.clone();
+        // assert!(ServiceRequest::from_request(r).is_err());
```

Comment:
Why is this commented out?

Reply:
With this change these calls would not fail. They lost the purpose of testing the failure case.

It's temporary disabled until the change proven to be working as intended.

We don't need rush to remove tests. It can be left there until everything settled down and if things goes south it can be revived very quickly.

Reply:
I'd keep it enabled or drop it. We risk possibly merging commented code, and there is no need for this as if we want to revert this, Git is our good friend. 👍🏼

Reply:
Merging comment code is nothing wrong. There are a lot of commented code through out tests and actix repos. You can make a PR to clean them up if you like to.

Reply:
At least add a todo marker.

Reply:
I think it is wrong to merge commented code as we have an SCM which allows the revert if need.

Reply:
OTOH, a TODO marker is searchable for outstanding items without making an issue in all commits until it is sorted. I find them very valuable.